com.civicrm.hrjobroles/CRM/Hrjobroles/Import/Parser/HrJobRoles.php

```diff
@@ -418,20 +427,19 @@ function fini() {}
 
 
   /**
-   * get the Option Value database value given its label
-   * @param String|Integer $option
+   * Get the (Option Value) database value given its
+   * label or the value itself
+   *
+   * @param String| $option
    * @param String $value
-   * @return String|NULL
+   * @return Integer|String|NULL
    * @access private
    */
   private function getOptionKey($option, $value)  {
-    $index = array_search(strtolower($value), array_column($this->_optionsList[$option], 'label'));
-    if ($index !== FALSE)  {
-      return $this->_optionsList[$option][$index]['value'];
-    }
-    else {
-      return NULL;
+    if (CRM_Utils_Array::value($value, $this->_optionsList[$option])){
+      return $value;
     }
+    return CRM_Utils_Array::key(strtolower($value), $this->_optionsList[$option]);
```

Comment:
I got a little lost here. Why does this method sometimes returns the value and sometimes returns the label? This seems different from the past version, which returned the value of null.